python/servo/try_parser.py

```diff
@@ -83,12 +83,10 @@ def handle_preset(s: str) -> Optional[JobConfig]:
         return JobConfig("MacOS", Workflow.MACOS, unit_tests=True)
     elif s in ["win", "windows"]:
         return JobConfig("Windows", Workflow.WINDOWS, unit_tests=True)
-    elif s in ["wpt", "linux-wpt"]:
-        return JobConfig("Linux WPT", Workflow.LINUX, unit_tests=True, wpt_layout=Layout.all())
     elif s in ["wpt-2013", "linux-wpt-2013"]:
         return JobConfig("Linux WPT", Workflow.LINUX, wpt_layout=Layout.layout2013)
-    elif s in ["wpt-2020", "linux-wpt-2020"]:
-        return JobConfig("Linux WPT", Workflow.LINUX, wpt_layout=Layout.layout2020)
+    elif s in ["wpt-2020", "linux-wpt-2020", "wpt", "linux-wpt"]:
+        return JobConfig("Linux WPT", Workflow.LINUX, unit_tests=True, wpt_layout=Layout.all())
```

Comment:
We only want to keep layout2020 here (without unit tests):

```suggestion
        return JobConfig("Linux WPT", Workflow.LINUX, wpt_layout=Layout.layout2020)
```

Reply:
Yes

Reply:
Do I also change `unit_tests` to `False` instead of `True`?
```
Running try_parser tests...
F...
======================================================================
FAIL: test_empty (python.servo.try_parser.TestParser.test_empty)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/tanishka/servo/python/servo/try_parser.py", line 185, in test_empty
    self.assertDictEqual(json.loads(Config("").to_json()),
AssertionError: {'fai[123 chars]ts': False, 'wpt_args': ''}, {'name': 'MacOS',[592 chars]''}]} != {'fai[123 chars]ts': True, 'wpt_args': ''}, {'name': 'MacOS', [591 chars]''}]}
Diff is 1401 characters long. Set self.maxDiff to None to see it.

----------------------------------------------------------------------
Ran 4 tests in 0.001s

FAILED (failures=1)
```

Reply:
yes